src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -429,19 +429,32 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator, IRBlobSe
                             case "!Plot":
                                 await _callbacks.Plot(
                                     new PlotMessage(
-                                        message.GetString(0, "xaml_file_path"),
-                                        message.GetInt32(1, "active_plot_index"),
-                                        message.GetInt32(2, "plot_count"),
+                                        message.GetString(0, "device_id").SafeParseGuid(),
```

Comment:
true, I think I added this across the board after I had issues with `graphics.ide.getactivedeviceid` where I sometimes returned NULL.  I could just leave it for the code that checks the result of `graphics.ide.getactivedeviceid` or also change that implementation to return empty guid instead of NULL.